charts/cluster-autoscaler/Chart.yaml

```
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.12.0
+version: 9.12.1
```

As we're adding new functionality rather than patching a bug, I'd prefer to make this a minor version bump.
```suggestion
version: 9.13.0
```

---

sure, I will do this later today

---

It's now Chart Version 9.13.0 and squashed